src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,42 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { User } from '../../types/User';
+import { Todo } from '../../types/Todo';
+import { getUser } from '../../api';
+
+type Props = {
+  handleClose: () => void;
+  selectedTodo: Todo | null;
```

Comment:
Don't forget about `selectedTodo?.` in your content
```suggestion
  selectedTodo: Todo;
```